src/components/Committees/CommitteesBanner/style.scss

```diff
@@ -0,0 +1,29 @@
+.committee-banner-container {
+	margin-bottom: 50px;
+	h1 {
+		margin: 20px 0 ;
+		font-size: 2.5em;
+	}
+
+	img {
+		box-shadow: none;
+	}
+
+    @media (min-width: 688px) {
+		min-height: 600px;
+		height: 50vh;
+
+		.banner-main-content {
+				position: relative;
+				top: 50%;
+				transform: perspective(1px) translateY(-50%);
```

Comment:
I'm unclear as to what this CSS does. I tried removing the relative position and the element looked like it rendered in the same way

Reply:
This just centers the image vertically in the parent container. For very large screens this has more of a noticeable effect; I've changed the way this is done in the new commit